DFC.ServiceTaxonomy.CustomFields/DFC.ServiceTaxonomy.CustomFields.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-rc1-12607" />
+    <PackageReference Include="OrchardCore.ContentManagement" Version="1.0.0-rc1-12607" />
+    <PackageReference Include="OrchardCore.ContentTypes.Abstractions" Version="1.0.0-rc1-12607" />
+    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.0.0-rc1-12607" />
+  </ItemGroup>
```

Comment:
Could you please add the Sonar analyzer to match the other projects:
```
    <PackageReference Include="SonarAnalyzer.CSharp" Version="8.6.0.16497">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
```

Happy for you to suppress the `S927` warning as we do elsewhere.